packages/parser-javascript/src/parser.ts

```diff
@@ -26,6 +42,40 @@ export default class JavascriptParser extends WebhintParser<ScriptEvents> {
             const ast = parse(sourceCode, options) as ESTree.Node;
             const tokens = [...tokenizer(sourceCode, options)];
 
+            /*
+             * We will need an array for each walk method supported.
+             */
+            const walkArray: WalkArray = [];
+
+            /*
+             * For now, only `walk.simple` is supported.
+             * When support for any other method is added we will need
+             * to reuse this method (in case of `ancestor`) or create
+             * a new one if the signature is not the same.
+             */
+            const simple = (node: ESTree.Node, visitors: NodeVisitor, base?: NodeVisitor, state?: any) => {
```

Comment:
Do we want to add support for the [other methods](https://github.com/acornjs/acorn/tree/master/acorn-walk#interface) while we are at it?

Reply:
ok

Reply:
@molant done!